README.md

```diff
@@ -1568,6 +1568,26 @@ type myBeautifulType = {
 }
 ```
 
+The same to Enum keys.
```

Comment:
I feel this is lost here. Maybe we could merge it in existing section, like: "For convention, use PascalCase for type names and enums values."

What do you think?

Reply:
Actually I prefer in this way. The text explain a different example to an existent section. If the texts is a unique block the reader can not note the difference in the second example.